cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -74,4 +74,71 @@ TEST(TestStringOps, TestCharLength) {
       << ctx.get_error();
 }
 
+TEST(TestStringOps, TestSubstring) {
+  gandiva::ExecutionContext ctx;
+  uint64_t ctx_ptr = reinterpret_cast<int64>(&ctx);
+  int32 out_len = 0;
+
+  char* out_str = substr_utf8_int64_int64(ctx_ptr, "asdf", 4, 1, 2, &out_len);
```

Comment:
can you please add a test for 0 len output ?

substr_utf8_int64_int64(ctx_ptr, "asdf", 4, 1, 0, &out_len)

Reply:
done